src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    margin: 0 4px 0 0;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:last-child {
+      margin: 0;
+    }
+  }
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
You need describe all variants ( for 1 to 5 stars, not only 4).
Try to use scss loop.